dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops

```diff
@@ -94,6 +94,31 @@ KUBERNETES_USER_API_RESPONSE_DEFAULT_USER_ATTR_NOT_FOUND=CWWKS5381W: The social
 KUBERNETES_USER_API_RESPONSE_DEFAULT_USER_ATTR_NOT_FOUND.explanation=The user API response does not contain a key that corresponds to the value of the configuration attribute that is specified in the message. The social login feature attempts to use a different key instead.
 KUBERNETES_USER_API_RESPONSE_DEFAULT_USER_ATTR_NOT_FOUND.useraction=Verify that the configuration attribute that is specified in the message is configured correctly. Ensure that the value of the configuration attribute corresponds to the entry in the user API response that should be used as the username.
 
+# 0=Exception message
+ERROR_INTROSPECTING_SERVICE_ACCOUNT=CWWKS5382E: The response from the service account user API cannot be processed: {0}
+ERROR_INTROSPECTING_SERVICE_ACCOUNT.explanation=Information about the service account cannot be obtained because another error occurred.
+ERROR_INTROSPECTING_SERVICE_ACCOUNT.useraction=If the message contains another error message, see the user action for that message. Otherwise, check the server logs for more error messages that might indicate where the other error occurred.
+
+# 0=HTTP response code, 1=Response body
+SERVICE_ACCOUNT_USER_API_BAD_STATUS=CWWKS5383E: The service account user API returned an unexpected [{0}] response code. Verify that the request to the API contains all of the required information and that the service account token is valid. The response from the API is [{1}].
+SERVICE_ACCOUNT_USER_API_BAD_STATUS.explanation=The service account user API did not return the expected status code.
```

Comment:
Please change "all of the required information" to "all required information" or, optionally, to "all the required information". This comment applies to the explanation and useraction.